src/redux/actions/billActions.js

```diff
@@ -52,3 +58,23 @@ export const fetchMyBills = (query_param = "") => {
     }
   };
 };
+
+export const updatePendingBill = (isAccepted, billId) => {
```

Comment:
and you confirmed that when `answerPendingBill` throws, `updatePendingBill` will also throw thus getting caught by whatever component calls it?

Reply:
yes, because it's within a try finally and not a try catch block